core/src/main/java/org/bitcoinj/core/BitcoinSerializer.java

```diff
@@ -286,7 +286,9 @@ private Message makeMessage(String command, int length, byte[] payloadBytes, byt
         } else if(command.equals("qsendrecsigs")) {
             return new QuorumSendRecoveredSignatures(params);
         } else if(command.equals("islock")) {
-            return new InstantSendLock(params, payloadBytes);
+            return new InstantSendLock(params, payloadBytes, InstantSendLock.ISLOCK_VERSION);
+        } else if(command.equals("isdlock")) {
+            return new InstantSendLock(params, payloadBytes, InstantSendLock.ISDLOCK_VERSION);
```

Comment:
this change allows us to receive both new and old InstantSend messages (`isdlock` is new)